cpg-core/src/main/kotlin/de/fraunhofer/aisec/cpg/graph/NodeBuilder.kt

```diff
@@ -269,3 +270,53 @@ fun <T : Node, S> T.codeAndLocationFrom(frontend: LanguageFrontend<S, *>, rawNod
 
     return this
 }
+
+fun <T : Node, S> T.codeAndLocationFromChildren(
```

Comment:
I think this method needs a good documentation here, because it is not self-explanatory that I need to specify the parent node (in raw AST form) to retrieve the code of its children.

Reply:
On it